hw/xbox/mcpx_apu.c

```diff
@@ -1526,6 +1597,11 @@ static void se_frame(void *opaque)
     }
 #endif
 
+    // Temporarily provide samples just from mixbin 0
+    qemu_spin_lock(&d->vp_out_buf_lock);
+    fifo8_push_all(&d->vp_out_buf, (uint8_t*)&mixbins[0][0], sizeof(mixbins[0]));
```

Comment:
Why not provide all samples here, so the recipient can run a proper GP / EP / ACI equivalent mixing?

Reply:
I mixed all samples into mixbin 0 here, so they are provided but this code path should be cleaned up